2nd-gen/test-functions-mocha/functions/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "functions",
+  "scripts": {
+    "lint": "eslint --ext .js,.ts .",
+    "build": "",
+    "serve": "npm run build && firebase emulators:start --only functions",
+    "shell": "npm run build && firebase functions:shell",
+    "start": "npm run shell",
+    "test": "mocha *.test.js",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log",
+    "compile": "cp ../../../tsconfig.template.json ./tsconfig-compile.json && tsc --project tsconfig-compile.json"
+  },
+  "engines": {
+    "node": "16"
+  },
+  "main": "lib/index.js",
```

Comment:
I think this should be `index.js` since there's no build (compile is just so we know the JS is valid)

Reply:
Updated!